### DISCUSSION TITLE
Crawled Property for Document Library

### DISCUSSION THREAD

**Discussion by aavi92** [microsoft-search/pnp-modern-search#2619] [Q&A] [answered]
Is there a Crawled Property for Document Library Name which can be used as a Filter ???

**Comment by kasperbolarsen** [comment, marked as answer]
On the road right now, so it is the short version: I would expect the name of the library is the same as Title. In order to use it as a refiner you will need to map Title to a refinablestring as Title is not sortable out of the box.

**Reply by aavi92** [reply]
Thanks for your response I have mapped Title to refinablestring however when i tried to add  RefinableString as Filter field & column value results are blank. Am i missing some thing ? 
![MSSTitle](https://user-images.githubusercontent.com/119278078/205207790-2de07683-c027-4d65-972d-463977f49c4b.png)

**Reply by aavi92** [reply]
![RFSProp](https://user-images.githubusercontent.com/119278078/205208147-d1972561-47ff-47db-bb38-b7bbba36fc19.png)

**Reply by kasperbolarsen** [reply]
You should expect that the mapping will take up to 24 hours to take effect.

**Reply by aavi92** [reply]
Thanks Kasper after 24 hours still no luck I have tried reindexing the site as well

**Reply by wobba** [reply]
See, this is a tricky one as `ows_Title` is not the right property to map for Title. You need to map `Office:2` and it should work :)

**Reply by kasperbolarsen** [reply]
Thanks @wobba , I knew that it was one of those "Basic:10, Office:2, Mail:5" but never found out which one in specific.

**Reply by aavi92** [reply]
Hi @wobba i have tried mapping office:2 and waited for more than 48 hours for crawl to happen no luck any other suggestions ? Thanks

**Comment by FranckyC** [comment]
You're right @kasperbolarsen, you can use the 'Title' property mapped with a RefinbleString.
